build.ps1

```diff
@@ -19,23 +19,32 @@ param (
     [switch]$Clean,
     # When specified, cargo test will be invoked for the specified component.
     [switch]$Test,
+    # When specified, cargo clippy will be invoked for the specified component.
+    [switch]$Lint,
+    # Run rustfmt
+    [switch]$Fmt,
     # When specified a build will not be run.
     [switch]$SkipBuild,
     # Use a optimized release build
     [switch]$Release,
     # Run a cargo check instead of build
     [switch]$Check,
-    # Add protocol feature
-    [switch]$protocols
+    # Features to pass to cargo
+    [string]$Features,
+    # Options to pass to the cargo test command
+    [string]$TestOptions,
```

Comment:
Is there any special quoting or anything that needs to be done here for multiple values?

Reply:
shouldn't be since it just emits the options as is. For example `-TestOptions "--test-threads=1"`